src/topology/algebra/module.lean

```diff
@@ -518,6 +604,10 @@ lemma comp_continuous_iff
   continuous (e ∘ f) ↔ continuous f :=
 e.to_homeomorph.comp_continuous_iff _
 
+/-- An extensionality lemma for `R ≃L[R] M`. -/
+lemma ext₁ [topological_space R] {f g : R ≃L[R] M} (h : f 1 = g 1) : f = g :=
```

Comment:
`@[ext]` perhaps? It can always be added later, or if you know a reason it's a bad idea to mark as `@[ext]` we should say why.

Reply:
We have a more general `ext` for `continuous_linear_equiv`. Can `ext` have two lemmas for the same type?